### ISSUE TITLE
Breaking change between version 12 and 13

### ISSUE THREAD

**Issue by HilSny** [tinycreative/react-native-intercom#281]
Hi!

We noticed our app crashing a lot after the recent upgrade (~5k crashes) and noticed that the readme changed in the 13.0 commit. If anyone else is having trouble, you'll have to remove

```
 <?xml version="1.0" encoding="utf-8"?>	
        <manifest package="com.myapp"	

           ...other configuration here...	

         >	
          <application	

             ...other configuration here...	

             xmlns:tools="http://schemas.android.com/tools"	
          >	

             <!-- ...other configuration here... -->	

             <service	
              android:name="com.robinpowered.react.Intercom.IntercomIntentService"	
              android:exported="false">	
              <intent-filter	
                android:priority="999">	
                  <action android:name="com.google.android.c2dm.intent.RECEIVE"/>	
              </intent-filter>	
            </service>	
            <receiver	
              android:name="io.intercom.android.sdk.push.IntercomPushBroadcastReceiver"	
              tools:replace="android:exported"	
              android:exported="true" />	
          </application>	
        </manifest>
```

Might be worth adding a note to the readme to indicate that this is the case if you're upgrading from 12 to 13.0. `react.Intercom.IntercomIntentService` no longer exists which causes the crashing (dex, class not found error).

**Comment by dani-mp** [comment]
Hi, @HilSny. Can you reproduce this in your testing environment?

I don't know if we should just remove that code or replace it with something else. Maybe some release notes or a list of breaking changes could help here.

**Comment by carlosa54** [comment]
Hi @HilSny,

This file was removed in favor of deprecating GCM. I made the change and didn't added a note to the readme. Thank you for the heads up

This link can give more context into why the file was removed. https://github.com/intercom/intercom-cordova/pull/251 

@browniefed Can we add the text below as a v13.0.0 release note?

### BREAKING CHANGE

* **AndroidManifest.xml** : Remove IntercomIntentService reference if you're using it in your manifest file
* Location: `android/app/src/main/AndroidManifest.xml`

Before:

```xml
<?xml version="1.0" encoding="utf-8"?>
<manifest package="com.myapp"
  ...other configuration here...
>
  <application
    ...other configuration here...
    xmlns:tools="http://schemas.android.com/tools"
  >
    <!-- ...other configuration here... -->
    <service
      android:name="com.robinpowered.react.Intercom.IntercomIntentService"
      android:exported="false">
      <intent-filter
        android:priority="999">
          <action android:name="com.google.android.c2dm.intent.RECEIVE"/>
      </intent-filter>
    </service>
    <receiver
      android:name="io.intercom.android.sdk.push.IntercomPushBroadcastReceiver"
      tools:replace="android:exported"
      android:exported="true" />
  </application>
</manifest>
```

After:

```xml
<?xml version="1.0" encoding="utf-8"?>
<manifest package="com.myapp"
  ...other configuration here...
>
  <application
    ...other configuration here...
    xmlns:tools="http://schemas.android.com/tools"
  >
    <!-- ...other configuration here... -->
    <receiver
      android:name="io.intercom.android.sdk.push.IntercomPushBroadcastReceiver"
      tools:replace="android:exported"
      android:exported="true" />
  </application>
</manifest>
```

**Comment by HilSny** [comment]
Sorry in the delay in replying. That would be great @carlosa54. Totally understand why it was removed, we just totally missed the config change which resulted in some crashing for our users so wanted to spread the word. Once we looked back at the code, we were able to figure it out. 

Thanks for maintaining a great library! This is the first time we've had any real issue with it and to be honest it was completely our fault :)
